cli/src/main/java/com/devonfw/tools/ide/tool/graalvm/GraalVm.java

```diff
@@ -67,8 +67,15 @@ public void uninstallPlugin(ToolPluginDescriptor plugin) {
     doGuPluginCommand(plugin, "remove");
   }
 
+  /**
+   * We only need to use gu if the graalvm community version is less than 21.0.0, gu was removed by the graalvm community edition and native-image was added by
+   * default now. See: <a href="https://www.graalvm.org/latest/reference-manual/graalvm-updater/">graalvm-updater</a>
+   */
   private void doGuPluginCommand(ToolPluginDescriptor plugin, String command) {
-    this.context.newProcess().errorHandling(ProcessErrorHandling.THROW_CLI).executable(getToolPath().resolve("bin").resolve("gu")).addArgs(command, plugin.name()).run(ProcessMode.DEFAULT);
+    if (this.getConfiguredEdition().equals("community") && this.getConfiguredVersion().isLess(VersionIdentifier.of("21"))) {
```

Comment:
This is incorrect: The default edition is `graalvm` and `community` edition is completely obsolete nonsense we just kept for `devonfw-ide` compatibility. Because of a bug in that `community` edition in very old version, you now disable the plugin installation for any edition other than `community` what is IMHO a bug rendering the plugin feature void.

If we want to address this bug properly why dont we check if `gu` is actually available instead of coding assumptions of editions and versions we just guessed.

Reply:
Replaced with your suggestion now.